tests/test_data/large/checksums.json

```diff
@@ -1,5 +1,5 @@
 {
-  "databento_mbo_xnas_itch.csv": "sha256:5481466e7776659f7f093b1ca8b05c7e88b8d586036692089cca5decc7965099",
+  "databento_mbo_xnas_itch.csv": "sha256:ac66f2775ebbfd71b5ac1e62f47df5d151150b18323eaeb4bcc88b330ee7a456",
```

Comment:
It's odd this checksum changed, probably a partial download unless Databento data has recently changed.
We're using it for an order book test which isn't fully completed, so not asserting anything.

I'll redownload myself after merge and see if the checksums have changed.

Reply:
I also get the new checksum after redownloading, so the data changed - which we expected might happen at some point.

Also highlights one of the benefits of these checksums.

Reply:
Looks like it was a partial or corrupted download, as I get the original checksum.